libraries/pipes/implementations/rust/src/lib.rs

```diff
@@ -281,6 +213,26 @@ mod tests {
                     (
                         "metadata".to_string(),
                         Some(json!({
+                            "int": {
+                                "raw_value": 100,
+                                "type": "int"
+                            },
+                            "float": {
+                                "raw_value": 100.0,
+                                "type": "float"
+                            },
+                            "bool": {
+                                "raw_value": true,
+                                "type": "bool"
+                            },
+                            "none": {
+                                "raw_value": null,
+                                "type": null
+                            },
+                            "timestamp": {
+                                "raw_value": 1000.0,
```

Comment:
Just checking to be sure, is this the expected format of a timestamp value?

Reply:
I believe so from inferring from `MetadataValue.timestamp`'s [source code](https://docs.dagster.io/_modules/dagster/_core/definitions/metadata/metadata_value#MetadataValue.timestamp) :)